src/pages/GameMasterBoard.js

```diff
@@ -67,3 +67,5 @@ const GameMasterBoard = () => {
 };
 
 export default GameMasterBoard;
+
```

Comment:
Nitpick, but can we remove all these random newlines from various files so we can see which files actually changed in this PR?